src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -16,17 +16,17 @@ public class MicrosoftIdentityOptions : OpenIdConnectOptions
         /// <summary>
         /// Gets or sets the Azure Active Directory instance, e.g. "https://login.microsoftonline.com".
         /// </summary>
-        public string Instance { get; set; }
+        public string Instance { get; set; } = null!;
```

Comment:
This is a workaround for properties that are really non-nullable, except in once instance when the Data Transfer Objects are not yet initialized.